### PR TITLE
Add chinese translations for AddPlugins things

### PR DESCRIPTION
**Pull request by BramChen**
* "updates" tab
* prompt for plugins that have subplugins

### CODE REVIEW

File: languages/zh-Hans/ControlPanel.multids [hunk 1]
```diff
@@ -16,7 +16,7 @@ Basics/NewJournal/Text/Prompt: 新日志条目的内容
 Basics/NewJournal/Title/Prompt: 新日志条目的名称
 Basics/NewTiddler/Tags/Prompt: 新条目的标签
 Basics/NewTiddler/Title/Prompt: 新条目的名称
-Basics/OverriddenShadowTiddlers/Prompt: 被覆写的默认条目数量
+Basics/OverriddenShadowTiddlers/Prompt: 被复写的默认条目数量
```
Comment (linonetwo) [inline languages/zh-Hans/ControlPanel.multids:19]: 复写 -> 覆盖

File: languages/zh-Hans/ControlPanel.multids [hunk 2]
```diff
@@ -111,8 +115,8 @@ Saving/GitService/GitLab/Password: 个人存取令牌的 API (详见 [[GitLab 
 Saving/GitService/Gitea/Caption: Gitea 保存模块
 Saving/GitService/Gitea/Password: 个人存取令牌的 API (通过 Gitea 的 Web 界面：`设定 | 应用程序 | 产生新的令牌`)
 Saving/TiddlySpot/Advanced/Heading: 高级设置
-Saving/TiddlySpot/BackupDir: 备份文件夹
-Saving/TiddlySpot/Backups: 备份
+Saving/TiddlySpot/BackupDir: 备分文件夹
```
Comment (linonetwo) [inline languages/zh-Hans/ControlPanel.multids:118]: 应该是「备份」啊

File: languages/zh-Hant/ControlPanel.multids [hunk 3]
```diff
@@ -83,6 +83,10 @@ Plugins/Plugins/Hint: 插件
 Plugins/Reinstall/Caption: 重新安裝
 Plugins/Themes/Caption: 佈景主題
```
Comment (linonetwo) [inline languages/zh-Hant/ControlPanel.multids:84]: 布景

File: languages/zh-Hant/ControlPanel.multids [hunk 4]
```diff
@@ -83,6 +83,10 @@ Plugins/Plugins/Hint: 插件
...
 Plugins/Themes/Hint: 佈景主題插件
+Plugins/Updates/Caption: 更新
+Plugins/Updates/Hint: 已安裝插件的可用更新
+Plugins/Updates/UpdateAll/Caption: 更新 <<upgrade-count>> 個插件
+Plugins/SubPluginPrompt: 提供 <<count>> 個可用的附加插件
 Parsing/Caption: 解析
 Parsing/Hint: 在此您可以全域停用或啟用維基解析規則。要使更改生效，請儲存並重新載入您的維基。停用某些解析規則，會妨礙 <$text text="TiddlyWiki"/>  正常運作。可使用[[安全模式|https://tiddlywiki.com/#SafeMode]]恢復正常操作。
```
Comment (linonetwo) [inline languages/zh-Hant/ControlPanel.multids:91]: 这个也还是繁体

Comment (BramChen) [reply languages/zh-Hant/ControlPanel.multids:91]: Thanks @linonetwo, zh-Hant 為繁體漢字無誤

File: languages/zh-Hant/ControlPanel.multids (hunk 3 above)
Comment (BramChen) [reply languages/zh-Hant/ControlPanel.multids:84]: Thanks @linonetwo, zh-Hant 為繁體漢字無誤

File: languages/zh-Hans/ControlPanel.multids (hunk 2 above)
Comment (BramChen) [reply languages/zh-Hans/ControlPanel.multids]: Thanks @linonetwo

File: languages/zh-Hant/ControlPanel.multids (hunk 4 above)
Comment (linonetwo) [reply languages/zh-Hant/ControlPanel.multids:91]: oh, 抱歉，我还以为你在翻译简体中文，我误会啦！

**Comment by linonetwo** [comment]
Please ignore my suggestion, I mistake this for simplified Chinese!

**Comment by BramChen** [comment]
@linonetwo, your feedbacks are always welcome.

**Comment by Jermolene** [comment]
Thanks @BramChen, and @linonetwo -- as @BramChen says, it's actually really good to get feedback on the translations. Keeping them up to date as carefully as @BramChen is a lot of work, and it's great to see it appreciated.

**Pull request merged**